### PR TITLE
[Filter] change prop name (in/out combi)

### PR DESCRIPTION
**Pull request by jaeyun-jung**
Change property name for in/out combination and update document.

Signed-off-by: Jaeyun <jy1210.jung@samsung.com>

### CODE REVIEW

**Review by gichan-jang — APPROVED** [review]
LGTM.
After this PR, related nnstreamer-example samples need to modified with this change.

File: gst/nnstreamer/tensor_filter/tensor_filter_common.c [hunk 1]
```diff
@@ -1785,7 +1785,7 @@ _gtfc_setprop_OUTPUTCOMBINATION (GstTensorFilterPrivate * priv,
     } else {
       ml_loge ("Wrong format for output combination properties. "
           "Please specify for input tensor(s): p#num, for output tensor(s): o#num "
-          "e.g., outputCombination=p0,p2,o0,o1");
+          "e.g., output-combination=p0,p2,o0,o1");
```
Comment (gichan-jang) [inline gst/nnstreamer/tensor_filter/tensor_filter_common.c:1788]: Need to chananged
```suggestion
          "e.g., output-combination=i0,i2,o0,o1");
```

**Review by dongju-chae — APPROVED** [review]
LGTM

File: gst/nnstreamer/tensor_filter/tensor_filter_common.c (hunk 1 above)
Comment (jaeyun-jung) [reply gst/nnstreamer/tensor_filter/tensor_filter_common.c]: updated. please see the changes again.

**Pull request merged**